types/migrations/20220428234639_add_morpheme_tag_description.sql

```diff
@@ -0,0 +1,36 @@
+-- Move the description column from abstract_morpheme_tag to morpheme_tag, so
+-- that morpheme tags in each system can have a distinct description.
+alter table morpheme_tag
+add column description text;
+
+update morpheme_tag
+set description = abstract_morpheme_tag.description
+from abstract_morpheme_tag
+where abstract_morpheme_tag.id = any(morpheme_tag.abstract_ids);
+
+alter table abstract_morpheme_tag
+drop column description;
+
+-- Add the Combine variant of SegmentType.
+alter type segment_type add value 'Combine';
+
+-- Allow tags to have a segment type override.
+alter table morpheme_tag
+add column segment_type segment_type;
```

Comment:
Fair enough, any suggestions for another name for the column or type?

Reply:
Something reflective of what the column represents, perhaps `role` or `segment_category`? To avoid saying the same thing twice while not being repetitive and going "morpheme_tag.morpheme_type".

Reply:
Yeah it's a functional distinction (i.e. "what is the function of this word part in context?") so `role` might work. Along those lines, the "Morpheme" variant may deserve a rename too.